Makefile

```diff
@@ -339,6 +355,12 @@ $(TEST_TARGETS): run-tests
 test-unit-cover: ARGS=-timeout=10m -race -coverprofile=coverage.txt -covermode=atomic
 test-unit-cover: TEST_PACKAGES=$(PACKAGES_UNIT)
 
+test-e2e: docker-build-e2e-init docker-build-local
+	INITIAL_VERSION=$(PRE_UPGRADE_VERSION) TARGET_VERSION=$(TARGET_VERSION) \
```

Comment:
Why do you use two different variables for the version before the upgrade (INITIAL_VERSION and RE_UPGRADE_VERSION)? Would only use one to avoid confusion.

Reply:
PRE_UPGRADE_VERSION depricated